lib/builder/components/ChartContainer.js

```diff
@@ -277,10 +281,8 @@ class ChartContainer extends Component {
                 />
               </div>
             )}
-          </>
-        )}
         <Chart
-          key={id}
```

Comment:
Is that change is necessary ? Using _index_ as a component _key_ could lead to strange behaviour and should be avoided.
You could read more about that [here](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318).